eventually/authentication/migrations/0002_customuser_user_profile.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-11-02 15:43
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('customprofile', '0001_initial'),
+        ('authentication', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='customuser',
+            name='user_profile',
+            field=models.OneToOneField(null=True, on_delete=django.db.models.deletion.CASCADE, to='customprofile.CustomProfile'),
```

Comment:
too long line. Is it generated automatically by a migrating script?

Reply:
Yes, it was generated automatically. Fixed. Made migrations again.